### DISCUSSION TITLE
[QUESTION] Deployment of app is down while it waits for replica rebuilding...

### DISCUSSION THREAD

**Discussion by migs35323** [longhorn/longhorn#4628] [Q&A]
## Question

So, an issue happened with 2 worker nodes which i had to restart, and now longhorn is rebuilding one missing replica from a deployment
The volume is huge, and now the app is stuck waiting for the longhorn to rebuild the missing replica..
this takes about an hour due to size, and my app is down meanwhile, is there something that i can do to prevent this situation?
i am currently schedulling just 2 replicas due to the size of the volume, creating a third one will allow me to keep the app working while the rebuild process happens?

## Environment

 - Longhorn version: 1.3.1
 - Kubernetes version: 1.24
 - Node config
   - OS type and version ubuntu 22.04
   - CPU per node: 4
   - Memory per node: 8
   - Disk type ssd

**Comment by c3y1huang** [comment]
No, you need to wait for the rebuild to finish.
